public/blog/announcing-full-portable-solidity-debugger.md

```diff
@@ -0,0 +1,79 @@
+# Announcing our Fully Featured, Portable Solidity Debugger
+
+Every one at Truffle is giddy right now. 
+
+----------------
+
+Since Truffle's inception, it's been our mission to build outstanding development tools for the Ethereum community. The analogy we always used was one of history: Ethereum development practices are years behind the rest of the industry, and it's our job to build tools that modernize our craft. Well today, Truffle's come one step closer to that goal. We're happy to announce the release of our fully featured, fully portable Solidity [debugger](/docs) and [debugging libraries](/docs). 
+
+    image or gif will live here
+
+### What We've Built
+
+If you've followed Truffle development, you're familiar with the `truffle debug` command. This was the first incarnation of a "debugger" built into Truffle, though it wasn't much more than a code tracer. Our new debugger comes with everything you expect :
+
+* Code stepping (over, into, out, next, instruction, etc.)
+* Current code location, including the address of the running contract
+* Breakpoints
+* Watch expressions
+* Variable inspection (stack, memory _and_ storage)
+* Custom expression evaluation using Solidity variables
+
+On top of that, our new debugger is fully portable, built to integrate with any Javascript project or editor, like VSCode. Lastly, we built it to work with **any Ethereum client**, so you could debug transactions against the main Ethereum network if you so desired. 
```

Comment:
Prefer the first mention to use its full name, "Visual Studio Code" ("VSCode" in parens if necessary).